pkg/apis/garden/v1beta1/types.go

```diff
@@ -758,6 +758,8 @@ type ShootStatus struct {
 	// UID is a unique identifier for the Shoot cluster to avoid portability between Kubernetes clusters.
 	// It is used to compute unique hashes.
 	UID types.UID `json:"uid"`
+	// KubernetesStatus contains the status of kubernetes specific parameters
+	KubernetesStatus KubernetesStatus `json:"kubernetes"`
```

Comment:
```suggestion
	Kubernetes KubernetesStatus `json:"kubernetes"`
```

Reply:
Adding new required field is not backwards-compatible change. Change it to optional and pointer.